### PR TITLE
Phoenix/project page

### LINKED ISSUE #331: Project system > project page
Create a new page that lives at `/projects/[id]`, where the id is the Firebase id associated with a project.

This page should emulate a very basic Devpost project page (an example [here](https://devpost.com/software/soup-kitchen-express)) and include (in rough order):
- project title
- short description
- names of team members (should be passed in as an array of strings)
- embedded YouTube video
- longer description
- links (should be passed in as an array of strings)

### PR DESCRIPTION
**Pull request by phoenixthefirebird**
image

## Description
Project pages residing at the path `/projects/[project_id]`, pulls info from firebase, no authentication needed to view the page.

image
image

@anneguo3 let me know if you and design want me to change anything about the UI 

closes #331

### CODE REVIEW

File: src/pages/ProjectView.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      </StyledBanner>
+      <P>{cutString(project.shortDescription, MAX_CHARACTERS_IN_DESCRIPTION)}</P>
+      <StyledYoutube src={project.youtubeUrl} />
+      <StyledDiv>
+        <StyledH2>Long Description</StyledH2>
+        <P>{project.longDescription}</P>
+      </StyledDiv>
+      <StyledDiv>
+        <StyledH2>Relevant Links</StyledH2>
+        <div>
```
Comment (kevin-zou) [inline src/pages/ProjectView.js:77]: For the relevant links div, can we apply a flexbox to it with flex-wrap and justify-content center? Keeps it responsive at small viewports

File: src/pages/ProjectView.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  align-items: center;
+  min-width: 100%;
+`
+
+const StyledYoutube = styled(Youtube)`
+  width: 600px;
+  height: 350px;
+  border-radius: 3px;
+`
+const StyledBanner = styled.div`
```
Comment (kevin-zou) [inline src/pages/ProjectView.js:33]: Can we add `text-align: center` to the banner styles?

**Review by kevin-zou — COMMENTED** [review]
This looks great - thanks for doing such a good job on it without any design input! Left a few comments on stuff, but should be pretty easy to address. Lmk if you have any questions :))

File: src/pages/ProjectView.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+      <StyledYoutube src={project.youtubeUrl} />
+      <StyledDiv>
+        <StyledH2>Long Description</StyledH2>
+        <P>{project.longDescription}</P>
+      </StyledDiv>
+      <StyledDiv>
+        <StyledH2>Relevant Links</StyledH2>
+        <div>
+          {project.links.map(link => {
+            const cleanedUpLink = link.replace('https://', '').replace('http://', '')
```
[the remark this answers is not in the document]
Comment (phoenixthefirebird) [reply src/pages/ProjectView.js:79]: Oh that makes sense!

File: src/pages/ProjectView.js (hunk 2 above)
Comment (phoenixthefirebird) [reply src/pages/ProjectView.js:37]: Added, didn't seem to change much for me tho?

Comment (kevin-zou) [reply src/pages/ProjectView.js:37]: If you view it at a viewport small enough to break either the title or team member list into two lines, it should be left-aligned (which is probably not what we want) - lmk if this shows up for you!

**Review by kevin-zou — APPROVED** [review]
Lgtm! Thanks for addressing my comments so quickly 🚀

File: src/pages/ProjectView.js (hunk 2 above)
Comment (phoenixthefirebird) [reply src/pages/ProjectView.js:37]: Oooh I see thanks!

**Pull request merged**